client/src/main.ts

```diff
@@ -44,21 +45,24 @@ Sentry.init({
   dsn: process.env.VUE_APP_SENTRY_DSN,
 });
 
-djangoRest.restoreLogin(store).then(async () => {
+(async () => {
+  // If user closes the tab, we want them to be logged out if they return to the page
```

Comment:
This is only the case if they have the tab closed for at most ten minutes, right? Is it desired that their token is immediately invalidated once they close the tab?

Reply:
I wanted to make this strictly less than the server-side auth duration (which is 30 minutes). Note that this code only invalidates the _oauth_ token, not the server-side login credentials. So if they return to the tab and their server-side token is still in the valid 30-minute window, they will go through the redirect loop to reacquire a new oauth token, but won't have to actually log in. To the user, it will just appear to take slightly longer to load the page after that 10 minutes but before 30.

Reply:
Ok, that makes sense. Thanks for explaining.